### PR TITLE
feat: added endpoint for getting total no of token/trees per continen…

### PR DESCRIPTION
**Pull request by Mohmn**
…t for a gived wallet id

context https://github.com/Greenstand/treetracker-web-map-client/pull/648#issuecomment-1170826460
Screenshot from 2022-07-02 14-38-25

### CODE REVIEW

File: __tests__/e2e/wallets.spec.ts [hunk 1]
```diff
@@ -31,4 +31,18 @@ describe('wallets', () => {
...
+    );
+    expect(response.status).toBe(200);
+    expect(response.body.walletStatistics).toBeInstanceOf(Array);
+    expect(response.body.walletStatistics).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({ continent: 'Africa' }),
+        expect.objectContaining({ continent: 'North America' }),
+      ]),
+    );
+  });
```
Comment (dadiorchen) [inline __tests__/e2e/wallets.spec.ts:47]: I think GET /wallets/:walletId/token-region-count is a better url if this, right? 
according to convention of restful, https://en.wikipedia.org/wiki/Representational_state_transfer the resource is representing: wallet resource [id] 's token count

Comment (dadiorchen) [reply __tests__/e2e/wallets.spec.ts:47]: The hierarchy is more reasonable I think

**Review by dadiorchen — CHANGES_REQUESTED** [review]

File: __tests__/e2e/wallets.spec.ts (hunk 1 above)
Comment (Mohmn) [reply __tests__/e2e/wallets.spec.ts:47]: yep URL is more structured as per your way

**Comment by dadiorchen** [comment]
@Mohmn good job, thanks!

**Pull request merged**